nilearn/plotting/displays.py

```diff
@@ -24,10 +30,10 @@
 # Local imports
 from .find_cuts import find_xyz_cut_coords, find_cut_slices
 from .edge_detect import _edge_map
-from ..image.resampling import get_bounds, reorder_img, coord_transform,\
-            get_mask_bounds
+from ..image.resampling import (get_bounds, reorder_img, coord_transform,
+                                get_mask_bounds)
```

Comment:
I usually try to refrain from making such cosmetic changes, if I am not touching the corresponding lines of code, because they make merging and rebasing hard.

Don't change it back, though.